### PR TITLE
Refactored JS files.

### PR DESCRIPTION
**Pull request by ivan-nginx**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [x] Bugfix.
- [x] Feature.
- [x] Code style update (formatting, local variables).
- [x] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): N/A

## What is the new behavior?
1. Refactored JS files with ESLint styles. `[refactoring]`
2. Moved internal descriptions of Tags to site links. `[refactoring]`
3. `label.js`: moved `trim()` from var to direct access. `[optimization]`
4. `tabs.js`: added `trim()` to `tabCaption` for avoid spaced icon with caption bug. `[bugfix]`
5. `exturl.js`: removed excess space from 'text' & move 'fa-external-link' icon from left to right. `[bugfix & change style]`
6. `full-image.js`: add `lazyload` & `size` control options; `lazy-image.js` tag no needed anymore `[2 features + duplicate fixed]`
7. `note.js`: added `alert` alias. `[feature]`
8. For now site link changed from Github to Website official link.

### How to use new `full-image.js` parameters?
In post content:
```md
{% fullimage /url [@lazy], [alt], [title], [size] %}
{% fi /url [@lazy], [alt], [title], [size] %}
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

File: scripts/tags/exturl.js [hunk 1]
```diff
@@ -48,19 +43,18 @@ function extURL(args, content) {
...
   var attrs = {
-    class: exturl,
+    class     : exturl,
     'data-url': url,
-    title: title
+    title     : title
   };
 
-  return htmlTag('span', attrs, text.join(' '));
+  return htmlTag('span', attrs, text.join(' ') + '<i class="fa fa-external-link"></i>');
```
Comment (maple3142) [inline scripts/tags/exturl.js]: IMO, this should be an option to let user decide.

Comment (ivan-nginx) [reply scripts/tags/exturl.js]: Emm.. can u add this option?

Comment (maple3142) [reply scripts/tags/exturl.js]: Maybe after this PR is merged into master.

Comment (ivan-nginx) [reply scripts/tags/exturl.js]: Ok.

**Pull request merged**